pkg/command/interfaces.go

```diff
@@ -12,13 +11,15 @@ type runner interface {
 	Stderr(err io.Writer)
 }
 
-type execRunner interface {
+// ExecRunner mock for intercepting calls to executables
+type ExecRunner interface {
 	runner
 	RunExecutable(e string, p ...string) error
-	RunExecutableInBackground(executable string, params ...string) (command.Execution, error)
+	RunExecutableInBackground(executable string, params ...string) (Execution, error)
```

Comment:
Minor: You could think of also moving the interface Execution into this file. 
The only reason that it is in a different file is that it had to be in pkg and this file was not.

Reply:
done.